models/PostsManager.php

```diff
@@ -28,10 +32,16 @@ public function listPosts() {
 		$req->closeCursor();
 
 		return $listPosts;
-
 	}
 
-	public function getPost($id){
+	/**
+	 *Récupère les information d'un post et les infos de l'auteur de ce post.
+	 *
+	 * @param int $id
+	 *
+	 * @return array
+	 */
+	public function getPost(int $id): array {
```

Comment:
Codacy found an issue: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)